src/System.Text.Json/src/System.Text.Json.csproj

```diff
@@ -19,6 +19,7 @@
     <Compile Include="System\Text\Json\JsonHelpers.cs" />
     <Compile Include="System\Text\Json\JsonHelpers.Date.cs" />
     <Compile Include="System\Text\Json\JsonTokenType.cs" />
+    <Compile Include="System\Text\Json\Serialization\Converters\JsonValueConverterGuid.cs" />
```

Comment:
nit: sort by alpha ordering next to the other `Converters`.

Reply:
Things I wish msbuild would do.